.github/workflows/docker-image-release.yml

```diff
@@ -29,19 +29,19 @@ jobs:
           images: |
             jetpackio/devbox
           tags: |
-            type=semver,pattern={{version}}
+            type=semver,pattern={{version}},value=${{ inputs.tag }}
           flavor: |
-            latest=${{ inputs.tag && true || false }}
+            latest=${{ !inputs.tag && true || false }}
```

Comment:
this is saying: latest is true if `inputs.tag` is falsy. Is that correct?

Do we need the `&& true` anymore? My guess is 'no'